ecosystem/models/test_results.py

```diff
@@ -1,10 +1,18 @@
 """Test results for commands."""
+from __future__ import annotations
 import datetime
-from typing import Optional, List
+from dataclasses import dataclass
+from enum import Enum
 
 from .utils import JsonSerializable
 
 
+class TestStatus(str, Enum):
```

Comment:
I don't think you need to subclass `str`? If you want to get the string out of the enum, use `my_enum.value`.